source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -150,62 +150,41 @@ Configure and run CRI-O + kata-runtime
 
        sudo systemctl daemon-reload
        sudo systemctl restart crio
-       sudo systemctl restart kubelet
 
 #.  Initialize the master control plane with the command:
 
     .. code-block:: bash
 
        sudo kubeadm init --cri-socket=/run/crio/crio.sock
 
-
 Install pod network add-on
 **************************
 
 You must choose and install a `pod network add-on`_ to allow your pods to
 communicate. Check whether or not your add-on requires special flags when you
 initialize the master control plane.
 
-**Notes about flannel add-on**
-
-If you choose the `flannel` add-on, then you must add the following to the
-`kubeadm init` command:
-
-..  code-block:: bash
-
-    --pod-network-cidr 10.244.0.0/16
-
-If you are using CRI-O and `flannel` and you want to use Kata Containers, edit the :file:`/etc/crio/crio.conf` file to add:
-
-..  code-block:: bash
-
-    [crio.runtime]
-    manage_network_ns_lifecycle = true
```

Comment:
The instructions above for Flannel are still needed. I would keep them.

What I meant is that the instructions regarding /opt/cni/bin can be removed from the Flannel section, not that the Flannel section itself can be removed.

Reply:
Review https://github.com/clearlinux/clear-linux-documentation/pull/393/commits/f67cf636d71687b0d02d1a371a6049af440021ea